cluster/addons/metadata-agent/stackdriver/metadata-agent.yaml

```diff
@@ -0,0 +1,38 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  labels:
+    app: stackdriver-agents
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+  name: stackdriver-agents
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: stackdriver-agents
+  template:
+    metadata:
+      labels:
+        app: stackdriver-agents
+    spec:
+      containers:
+      - image: us.gcr.io/container-monitoring-storage/stackdriver-metadata-agent:{{ metadata_agent_version }}
+        imagePullPolicy: IfNotPresent
+        name: metadata-agent
+        ports:
+        - containerPort: 8000
```

Comment:
nit: as this rolls into production, we will want to look at occupying a less common port.

Reply:
Thanks for pointing this out, I'll change it in a follow-up PR. Do we need to agree on some port number (e.g. for fluentd to communicate), or it doesn't make a difference for you now?